language-server/vscoqtop/lspManager.ml

```diff
@@ -350,6 +350,19 @@ let coqtopLocate ~id params =
     let result = Ok (`String str) in
     output_json @@ Response.(yojson_of_t { id; result })
 
+let coqtopPrint ~id params = 
+  let open Yojson.Safe.Util in
+  let textDocument = params |> member "textDocument" in
+  let uri = textDocument |> member "uri" |> to_string in
+  let loc = params |> member "position" |> parse_loc in
+  let pattern = params |> member "pattern" |> to_string in 
+  let st = Hashtbl.find states uri in
+  match Dm.DocumentManager.print st loc ~pattern with
+  | Error _ -> ()
```

Comment:
Shouldn't we return an error response here?

Reply:
Yes, but we need to handle the error management anyways in most the other queries as well... I think we should make a separate issue and do it all in one go !